### ISSUE TITLE
Edit User Profile

### ISSUE THREAD

**Issue by rahulgarg97** [gcivil-nyu-org/nyc-accessible-restaurant-advisor#124]
Issue tracker is ONLY used for reporting bugs. New features should be discussed in its own thread on the #general channel of slack.

## Expected Behavior
When a User edits their profile, they can enter anything in any of the fields. There are no checks and balances, what a user enters in a field except for the username field

## Current Behavior
It just allows users to put anything in edit profile fields

## Ideas for Improvement

Checks like for phone no. (at least, verify the length to be 10 digits, entered letters should be digits only), for the city (the letters entered should not be digits), for zip (letters can only be digits and of length 5), for the state (it can only be alphabets, no digits should be allowed and maybe allowing only valid US states)

## Steps to Reproduce

1. After login, just go to the edit profile, and try to put anything, in the fields except username
2.
3.
4.

## Context (Environment)

Basically, maybe in the future, the user information like how many users are from the same city using this website, how many users live in a specific zipcode, etc. can be useful for the company. Even say, for recommending restaurants, based of collected data, you can recommend restaurants based on their addresses.
